code/modules/mining/equipment/angle_grinder.dm

```diff
@@ -0,0 +1,94 @@
+/*
+ * YES it's tweaked chainsaw code, do something about it.
+*/
+
+/*
+ * ############## Places where TOOL_DECONSTRUCT is used ############
+ *
+ * code/game/atoms.dm (for the definition of deconstruct_act(mob/living/user, /obj/item/I)
+ *
+ * code/game/objects/structures/salvaging.dm (waste planet salvagable machines)
+ *
+ * code/game/turfs/closed/walls.dm
+ * code/game/objects/structures/girders.dm
+ * code/game/objects/structures/grille.dm
+ * code/game/objects/structures/catwalk.dm
+ * code/game/objects/structures/lattice.dm
+ * code/game/objects/structures/crates_lockers/closets.dm
+ * code/game/objects/structures/tables_racks.dm (both normal and reinforced)
+ * code/game/objects/structures/tables_frames.dm
+ * code/game/objects/structures/window.dm (reinf window)
+ * code/game/objects/structures/bed_chairs/bed.dm
+ * code/game/objects/structures/bed_chairs/chair.dm
+*/
+
+/*
+ * Configure features by editing __DEFINES/anglegrinder.dm
+*/
+
+/obj/item/anglegrinder
+	name = "angle grinder"
+	desc = "A powerfull tool to salvage about anything. You better wear some protection tough"
```

Comment:
minor spelling/grammar mistakes. "A powerful tool to salvage just about anything, you better wear some protection though."

Preferably, I would replace that second part with something that outright states what is protection. "...anything, you should wear earmuffs and welding protection to avoid damage to your ears and eyes." would be my best guess but something longer might be better.